### DISCUSSION TITLE
reserved identifier violation [8615]

### DISCUSSION THREAD

**Discussion by elfring** [eProsima/Fast-DDS#3457] [Ideas]
I would like to point out that identifiers like “[`_FASTRTPS_CONFIG_H_`](https://github.com/eProsima/Fast-RTPS/blob/a9401420fbab28b075e765d602c1caaed30decce/include/fastrtps/config.h.in#L15 "Update candidate")” and “[`_UTILS_TIMEDCONDITIONVARIABLE_HPP_`](https://github.com/eProsima/Fast-RTPS/blob/e141f121aa20263e96d3390c8ee7969422620f0f/include/fastrtps/utils/TimedConditionVariable.hpp#L19 "Another update candidate")” [do eventually not fit](https://www.securecoding.cert.org/confluence/display/cplusplus/DCL51-CPP.+Do+not+declare+or+define+a+reserved+identifier#DCL51-CPP.Donotdeclareordefineareservedidentifier-NoncompliantCodeExample%28HeaderGuard%29 "Do not declare an identifier which is reserved for the compiler implementation.") to the expected naming convention of the C++ language standard.
Would you like to adjust your selection for unique names?

**Comment by JLBuenoLopez** [comment]
While the report is correct, the risk assessment and the likelihood of happening is low, so for the moment is not going to be fixed. I am labeling this as enhancement and as per [Fast DDS Contributing guidelines](https://github.com/eProsima/Fast-DDS/blob/master/CONTRIBUTING.md#issues-and-support) moving it to the Ideas discussion forum.
